gen/cheader.tmpl

```diff
@@ -262,6 +263,13 @@ typedef struct WGPUStringView {
     size_t length;
 } WGPUStringView;
 
+/**
+ * Sentinel value used in @link WGPUStringView to indicate that the pointer
+ * is to a null-terminated string, rather than an explicitly-sized string.
+ */
+#define WGPU_STRLEN
```

Comment:
This should be `#define WGPU_STRLEN SIZE_MAX` ?

Reply:
Oops ^_^ thank you!